browser/resources/settings/brave_leo_assistant_page/model_config_ui.ts

```diff
@@ -166,37 +165,37 @@ export class ModelConfigUI extends ModelConfigUIBase {
   }
 
   onModelLabelChange_(e: any) {
-    this.label = e.target.value
+    this.label = e.value
   }
 
   onModelRequestNameChange_(e: any) {
-    this.modelRequestName = e.target.value
+    this.modelRequestName = e.value
   }
 
   onModelSystemPromptChange_(e: any) {
-    this.modelSystemPrompt = e.target.value
+    this.modelSystemPrompt = e.value;
   }
 
   onContextSizeChange_(e: any) {
-    this.contextSize = parseInt(e.target.value, 10);
+    this.contextSize = parseInt(e.value, 10);
```

Comment:
optional: You might be able to use `e.valueAsNumber` here (I think it'll be `NaN` for bad parses too)

Reply:
Good call! I forgot about that little feature😀